plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/FilesTable.java

```diff
@@ -85,8 +85,8 @@ public FilesTable(SchemaTableName tableName, TypeManager typeManager, Table iceb
                         .add(new ColumnMetadata("lower_bounds", typeManager.getType(mapType(INTEGER.getTypeSignature(), VARCHAR.getTypeSignature()))))
                         .add(new ColumnMetadata("upper_bounds", typeManager.getType(mapType(INTEGER.getTypeSignature(), VARCHAR.getTypeSignature()))))
                         .add(new ColumnMetadata("key_metadata", VARBINARY))
-                        .add(new ColumnMetadata("split_offsets", new ArrayType(BIGINT)))
-                        .add(new ColumnMetadata("equality_ids", new ArrayType(INTEGER)))
+                        .add(new ColumnMetadata("split_offsets", typeManager.getType(arrayType(BIGINT.getTypeSignature()))))
```

Comment:
What's the difference between `new ArrayType(BIGINT)` (old) and `typeManager.getType(arrayType(BIGINT.getTypeSignature()))` (new)?

Reply:
I think they are the same(please correct me if I'm wrong). The only reason that I changed to use `typeManager` is that I want to follow the similar semantic(e.g. `mapType`).